src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -66,7 +66,7 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
         final Pair<Double, Double> concordance = calculateConcordance(filteredVcf, truthVcf);
         final double sensitivity = concordance.getLeft();
         final double fdr = concordance.getRight();
-        Assert.assertTrue(sensitivity > requiredSensitivity);
+        Assert.assertTrue("Sensitivity " + sensitivity + " was lower than required sensitivity of " + requiredSensitivity, sensitivity > requiredSensitivity);
```

Comment:
Nope.  I was just as surprised.  But I saw the failures working properly during my debugging.